uat/features/mollom.feature

```diff
@@ -15,7 +16,8 @@ Feature: Mollom
     Then I should see "The form protection has been added"
 
 # This will fail until we figure out a way to pass private key
+  @api @dev @destructive
```

Comment:
I like that idea but I think that may bring out the lazy side in me and instead of fixing issues I may just tag things as know to fail.